dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -865,7 +871,7 @@
 	 * 2. The host id
 	 * 3. The host id
 	 */
-	private final String deleteStructurePermissionsByPathSQL =
+	private static final String deleteStructurePermissionsByPathSQL =
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)